WooCommerce/Classes/Authentication/Epilogue/StorePickerViewController.swift

```diff
@@ -220,6 +226,15 @@ private extension StorePickerViewController {
                                                            action: #selector(cleanupAndDismiss))
     }
 
+    func setupNavigationForListOfConnectedStores() {
+        title = NSLocalizedString("Connected Stores", comment: "Page title for the list of connected stores")
```

Comment:
Should this be inside a `Localizations` enum?

Reply:
Right. I didn't do it because this class does not declare a Localizations enum, and I didn't want to change anything that wasn't strictly necessary. I would prefer to leave this as it is for now. IMO, we really need to rethink StorePickerViewController and break it down.

Reply:
⚠️ Does this means we now have an entirely new string in the codebase after this PR? This means that this string won't be translated in time for 5.8 – as the GlotPress bundle containing all the copies to be translated has already been sent to our translations vendor on Sunday